cluster/gce/upgrade.sh

```diff
@@ -580,6 +591,8 @@ if [[ "${node_upgrade}" == "true" ]]; then
   fi
 fi
 
+update-coredns-config
```

Comment:
Actually, mind putting a comment explaining why this step is needed?

Reply:
Done!